app/models/user.rb

```diff
@@ -104,8 +104,8 @@ def student
 
   validates :name, :email, :country, :location, presence: true, unless: :github_import
 
-  accepts_nested_attributes_for :attendances, allow_destroy: true
-  accepts_nested_attributes_for :roles, allow_destroy: true
+  accepts_nested_attributes_for :attendances, allow_destroy: true, limit: 3
+  accepts_nested_attributes_for :roles
```

Comment:
The max of conferences is 3 _per raffle round_. That's why I wanted to set the limit on the `nested_attributes`: so it only applies on form `submit` for the relevant raffle round. I suspect that either solution above doesn't work on the form only?
I can use either the `limit` option or the `reject_if` option and add a Proc or a symbol. So I am thinking something like this: 
```
accepts_nested_attributes_for :attendances, allow_destroy: true, 
reject_if: :too_many_attendances

MAX_ATTENDANCES_PER_ROUND = 3

def too_many_attendances
    this_round_attendances = attendances.collect {
      |attendance| attendance.conference.in?(Conference.in_current_season)
    }.count
    this_round_attendances > MAX_ATTENDANCES_PER_ROUND
  end
```
 NB `round` is an attribute of `conference`. I didn't take that into account yet, just checking if `option + method` is a viable way.

Is this a viable way?